library/common/extensions/cert_validator/platform_bridge/platform_bridge_cert_validator.cc

```diff
@@ -123,49 +125,69 @@ void PlatformBridgeCertValidator::verifyCertChainByPlatform(
   if (!success) {
     error_details = "PlatformBridgeCertValidator_verifySubjectAltName failed: SNI mismatch.";
```

Comment:
Is this SAN or SNI mismatch? The error message says SNI but the type is `FAIL_VERIFY_SAN`

Reply:
I think it's both :) It's saying that the SNI from the ClientHello (a single hostname) is not contained in the SAN from the certificate (list of hostname). The current code uses "SNI" in the error message and updates the "fail_verify_san_" stat. I preserved this behavior in this PR, as it's not intended to change behavior.

But it would be trivial to rename FAIL_VERIFY_SAN to FAIL_VERIFY_SNI or change the string in the error message if that would be clearer? (I suspect changing the name of the stat is out of scope).

Reply:
No need to rename, I just wanted to flag in case this was a typo or they meant different things.